triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -74,8 +76,8 @@ abstract class DbTriggerDao protected (
   // `Fragment.const` which will try to use a raw string as a SQL query.
 
   private def insertRunningTrigger(t: RunningTrigger): ConnectionIO[Unit] = {
-    val insert: Fragment = sql"""
-        insert into running_triggers
+    val insert: Fragment =
+      sql"insert into " ++ Fragment.const(s"${tablePrefix}running_triggers") ++ sql"""
```

Comment:
```suggestion
      sql"""insert into ${Fragment.const(s"${tablePrefix}running_triggers")}
```

Reply:
That looks much nicer, thank you! I fixed the rest.